app/models/version.rb

```diff
@@ -411,6 +411,10 @@ def gem_file_name
     "#{full_name}.gem"
   end
 
+  def pushed_by_trusted_publisher?
+    pusher_api_key&.trusted_publisher? ? true : false
```

Comment:
Doesn't `trusted_publisher?` already return true/false? Is there any reason for explicit `? true: false`?

Reply:
The safe navigator means a nil value can be returned if the version has no `pusher_api_key`.

Reply:
Would `pusher_api_key? && pusher_api_key.trusted_publisher?` works also?

Reply:
Or maybe `pusher_api_key&.trusted_publisher? || false`. Yes, just a nitpick indeed.

Reply:
Am I the only one that prefers `!!` to force boolean?